web/package.json

```diff
@@ -51,73 +53,55 @@
   },
   "scripts": {
-    "start": "CI=true craco start",
-    "start:cypress": "export $(cat .env.cypress | xargs) && craco --max_old_space_size=4096 -r @cypress/instrument-cra start",
-    "build": "craco --max_old_space_size=2048 build",
-    "build:prod": "craco --max_old_space_size=2048 build && yarn sentry:sourcemaps",
-    "test": "craco test --transformIgnorePatterns \"node_modules/(?!ol)/\"",
+    "dev": "vite",
+    "preview": "vite preview",
+    "dev:cypress": "export $(cat .env.cypress | xargs) && yarn dev",
+    "build:prod": "tsc -b && vite build",
+    "test": "vitest",
+    "coverage": "yarn run cypress:ci && yarn run coverage:ci && yarn run finalizeCoverage",
     "test:ci": "CI=true npm test",
-    "coverage": "npm test -- --coverage --watchAll=false",
-    "coverage:ci": "CI=true REACT_APP_KEYCLOAK_CLIENT=wps-web npm test -- --coverage --watchAll=false",
+    "coverage:ci": "CI=true VITE_KEYCLOAK_CLIENT=wps-web vitest run --coverage",
     "cy:open": "cypress open",
     "cy:run": "cypress run --browser chrome --config watchForFileChanges=false",
-    "cypress": "start-server-and-test start:cypress 3030 cy:open",
-    "cypress:ci": "start-server-and-test start:cypress 3030 cy:run",
-    "eject": "react-scripts eject",
+    "cypress:and:server": "start-server-and-test start:cypress 3030 cy:open",
+    "cypress:ci": "start-server-and-test dev:cypress http://localhost:3030 'cy:run'",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
-    "finalizeCoverage": "node mergeCoverage.js",
-    "sentry:sourcemaps": "sentry-cli sourcemaps inject --org bcps-wps --project frontend ./build && sentry-cli sourcemaps upload --org bcps-wps --project frontend ./build"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "finalizeCoverage": "yarn node mergeCoverage.cjs"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@craco/craco": "^7.1.0",
     "@cypress/code-coverage": "^3.10.0",
     "@cypress/instrument-cra": "^1.4.0",
     "@eslint/compat": "^1.1.1",
+    "@sentry/vite-plugin": "^2.22.2",
     "@testing-library/dom": "^10.1.0",
     "@testing-library/jest-dom": "^6.4.2",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/jest": "^29.5.12",
     "@types/recharts": "^1.8.23",
-    "cypress": "^13.0.0",
+    "@vitejs/plugin-react": "^4.3.1",
+    "cypress": "^13.13.3",
     "eslint": "^9.7.0",
     "eslint-plugin-react-hooks": "^4.6.2",
+    "eslint-plugin-react-refresh": "^0.4.7",
     "globals": "^15.8.0",
-    "react-scripts": "^5.0.1",
     "start-server-and-test": "^2.0.0",
-    "ts-sinon": "^2.0.2"
-  },
-  "jest": {
-    "collectCoverageFrom": [
```

Comment:
This is also probably why coverage dropped, does vitest not have a similar configuration option?

Reply:
yep it's in vitest.config test/coverage